packages/instant/src/containers/selected_asset_buy_order_state_buttons.ts

```diff
@@ -68,7 +70,11 @@ const mapDispatchToProps = (
         dispatch(actions.updateBuyOrderState({ processState: OrderProcessState.FAILURE, txHash })),
     onSignatureDenied: (buyQuote, error) => {
         dispatch(actions.resetAmount());
-        dispatch(actions.setError(error));
+        errorUtil.errorFlasher.flashNewError(dispatch, error);
```

Comment:
This will have conflicts with https://github.com/0xProject/0x-monorepo/pull/1187-- will have to update whichever branch gets merged second